webpack.config.js

```diff
@@ -116,9 +116,10 @@ const federationConfig = {
       filename: "remoteEntry.js",
       exposes: {
         "./Dashboard": "./src/entries/Dashboard.jsx",
-        "./DrugChartModal": "./src/entries/DrugChartModal/DrugChartModal.jsx",
-        "./DrugChartModalNotification":
-          "./src/entries/DrugChartModal/DrugChartModalNotification.jsx",
+        "./DrugChartSlider":
```

Comment:
the common components should come inside src/component folder, if we feel that it would be utilised at multiple places then it should be inside it otherwise inside features/DisplayControl/ folder